nucliadb/src/nucliadb/ingest/orm/brain.py

```diff
@@ -229,12 +229,20 @@ def apply_field_metadata(
 
     def delete_metadata(self, field_key: str, metadata: FieldComputedMetadata):
         for subfield, metadata_split in metadata.split_metadata.items():
+            self.brain.paragraphs_to_delete.append(
+                ids.FieldId(rid=self.rid, field_id=field_key, subfield_id=subfield).full()
+            )
+            # TODO: Bw/c, remove this when the new paragraph deletion by field
```

Comment:
what is paragraph deletion by field?
Isn't that what we have implemented in this pr?

Reply:
Yeah, maybe the new is not the best word. But yes, is the one implemented in this PR